tests/test_interrupted_sync.py

```diff
@@ -62,26 +64,45 @@ def get_bookmark_value(self, state, stream):
         return None
 
     ##########################################################################
-    # Tests To Override
+    # Tests and Functions To Override
     ##########################################################################
 
-    def test_bookmarked_streams_start_date(self):
-        # TODO - BUG - completed streams are not respecting the bookmark value
-        def override_lookback_window(stream=None):
-            look_back = {stream: timedelta(days=0) for stream in self.streams_to_test()}
-            if stream:
-                return look_back[stream]
-            return look_back
-
-        self.expected_lookback_window = override_lookback_window
-        super().test_bookmarked_streams_start_date()
-
-    def test_resuming_sync_records(self):
-        # TODO - BUG - completed streams are not respecting the bookmark value
-        def override_lookback_window(stream=None):
-            look_back = {stream: timedelta(days=0) for stream in self.streams_to_test()}
-            if stream:
-                return look_back[stream]
-            return look_back
-        self.expected_lookback_window = override_lookback_window
-        super().test_resuming_sync_records()
+    def calculate_expected_sync_start_time(self, bookmark, stream, completed=True):
+        """This method is only for streams that have bookmarks and a sync has been started"""
+
+        # BUG override bookmark to use final state vs manipulate_state allowing test to pass
+        done_status_list = [ "Closed", "Done", "Rejected" ]
+        if not GA4InterruptedSyncTest.jira_status:
+            GA4InterruptedSyncTest.jira_status = base.get_jira_card_status('TDL-23687')
```

Comment:
Sorry I forgot to point it out.

Need to fix this as well. 

```suggestion
            GA4InterruptedSyncTest.jira_status = get_jira_card_status('TDL-23687')
```

Reply:
Done.